omise/controllers/front/payment.php

```diff
@@ -0,0 +1,28 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . '/omise/classes/charge.php';
+}
+
```

Comment:
@nimid for the delegate class design at `omise/classes/charge.php`
I think we can more separate their responsibilities by keep `omise/classes/charge.php` do only connect with Omise API without know how to gather data from a form.

It means, `omise/classes/charge.php` doesn't need to extends `ModuleFrontController` and let `OmisePaymentModuleFrontController` pass the values that this class gather from a checkout form by itself, since this class only the class that directly communicate with frontend.